svc_queuedprocesses.go

```diff
@@ -0,0 +1,118 @@
+package lokalise
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+)
+
+const (
+	pathQueuedProcesses = "processes"
+)
+
+type QueuedProcessService struct {
+	BaseService
+}
+
+// ‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+// Service entity objects
+// _____________________________________________________________________________________________________________________
+
+type QueuedProcessType string
+
+const (
+	FileImport   QueuedProcessType = "file-import"
+	SketchImport QueuedProcessType = "sketch-import"
+)
+
+func (qpt *QueuedProcessType) UnmarshalJSON(b []byte) error {
+	// Define a secondary type to avoid ending up with a recursive call to json.Unmarshal
```

Comment:
You should not validate a server response in this case, a nice hack tho

Reply:
Updated